Resources/public/css/theme/views/loginform.css

```diff
@@ -24,8 +24,9 @@
 }
 
 .ez-view-loginformview .ez-loginform-title {
-    font-size: 0.9rem;
+    font-size: 0.95rem;
     color: #444;
+    font-family: sans-serif;
```

Comment:
and this is why it's the font-family style since almost day 1 https://github.com/ezsystems/PlatformUIBundle/commit/ce186bc96783b69b453c65fa863c78e8eb731350#diff-82166add900f62398a221f90e2ddd9c8R46